src/rest/SearchEndpoint.ts

```diff
@@ -1097,3 +1104,66 @@ function includeActionsHistory(historyStore: CoveoAnalytics.HistoryStore = new h
     return descriptor;
   };
 }
+
+function includeReferrer() {
```

Comment:
Actually this refers to the last page that the user visited. See `originLevel3` in the UA calls for a comparison.